docs/supported-browsers.csv

```diff
@@ -0,0 +1,5 @@
+Browser,Operating System,Versions
+Chrome,"Windows, MacOS, Linux, Android",>=90
+Firefox,"Linux, Windows",>=90
+Edge,Windows,>=90
+Safari,"MacOS, iOS",>=13
```

Comment:
I see no IE here, this makes me happy 👍

Reply:
Yeah, this is the change that I was describing. We are still testing IE11, but it is not supported and receives no new features.